app/build.gradle

```diff
@@ -1,26 +1,29 @@
-apply plugin: 'com.android.application'
+apply plugin: "com.android.application"
+apply plugin: "kotlin-android"
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    compileSdkVersion buildConfig.compileSdk
 
     defaultConfig {
-        applicationId "co.infinum.complexify_android"
-        minSdkVersion 14
-        targetSdkVersion 25
+        applicationId "com.infinum.complexify.sample"
+        minSdkVersion buildConfig.minSdk
+        targetSdkVersion buildConfig.targetSdk
         versionCode 1
         versionName "1.0"
```

Comment:
@JonatanPlesko Check how to add a releaseConfig for further optimisation. Refer to any of my libs until I update template repo for libs. Just a suggestion, not a mandatory change.